aqandu/api_routes.py

```diff
@@ -321,8 +519,44 @@ def request_model_data_local(lat, lon, radius, start_date, end_date):
     if query_job.error_result:
         print(query_job.error_result)
         return "Invalid API call - check documentation.", 400
-    rows = query_job.result()  # Waits for query to finish
+    sensor_data = query_job.result()  # Waits for query to finish
+
+    return(sensor_data)
+
 
+# could do an ellipse in lat/lon around the point using something like this
+#WHERE SQRT(POW(Latitude - @lat, 2) + POW(Longitude - @lon, 2)) <= @radius
+#    AND time > @start_date AND time < @end_date
+#    ORDER BY time ASC
+# Also could do this by spherical coordinates on the earth -- however we use a lat-lon box to save compute time on the BigQuery server
+
```

Comment:
If this is an improvement that you're considering adding, you should remove this code and open an issue on github to track it.

Reply:
Done.  That will get straighten out when I do the next commit.  Generally, I will fix my style to have few inline comments of this sort.  Figure that will take some time.  I would not hold up the pull on that basis, since the code that is currently live doesn't function properly.

Reply:
Done